x/pylons/keeper/grpc_query_get_stripe_refund_test.go

```diff
@@ -0,0 +1,72 @@
+package keeper_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/Pylons-tech/pylons/x/pylons/types"
+)
+
+func (suite *IntegrationTestSuite) TestGetStripeRefund() {
+	goCtx := sdk.WrapSDKContext(suite.ctx)
+
+	correctAddr := "pylo1xn72u3jxlpqx8tfgmjf0xg970q36xensjngsme"
+	amount := sdk.NewIntFromUint64(10020060)
+	productID := "recipe/Easel_CookBook_auto_cookbook_2022_06_14_114716_442/Easel_Recipe_auto_recipe_2022_06_14_114722_895"
+	signature := "8lZsKTOdMuJSoFn0RCGEUGpPXl4YzLmhJMrEiAd4qZh99S4IIGbvcsXyOcOHdlKi6Yys9NhmkLN4LqSlq8Y1Cw=="
```

Comment:
how do you get this signature?

Reply:
I get from other test. In this test, this signature use to create the message for StripeRefund

> https://github.com/Pylons-tech/pylons/blob/main/x/pylons/keeper/msg_server_add_stripe_refund_test.go

Reply:
:v, just hate this kind of arbitrary signature.